feature/ble/src/main/java/no/nordicsemi/android/wifi/provisioner/ble/view/BleProvisioningScreen.kt

```diff
@@ -76,22 +70,42 @@ fun BleProvisioningScreen() {
 
     val state by viewModel.state.collectAsStateWithLifecycle()
     val onEvent: (ProvisioningViewEvent) -> Unit = { viewModel.onEvent(it) }
-    Column {
-        NordicAppBar(
-            text = stringResource(id = R.string.label_ble_provisioner),
-            actions = {
-                LoggerAppBarIcon(
-                    onClick = { viewModel.onEvent(OpenLoggerEvent) }
-                )
-            },
-            showBackButton = true,
-            onNavigationButtonClick = viewModel::navigateUp
-        )
-        Box(modifier = Modifier.weight(1f)) {
-            Content(state) { viewModel.onEvent(it) }
+    Scaffold(
+        contentWindowInsets = WindowInsets(0, 0, 0, 0),
+        topBar = {
+            NordicAppBar(
```

Comment:
Nordic App Bar title should default to Single Line and ellipsize text which has to be fixed in commons library.